response_operations_ui/controllers/message_controllers.py

```diff
@@ -3,14 +3,36 @@
 import requests
 from flask import current_app, session
 from flask_login import current_user
-from requests.exceptions import HTTPError
+from requests.exceptions import HTTPError, RequestException
+from json import JSONDecodeError
 from structlog import wrap_logger
 
 from response_operations_ui.exceptions.exceptions import ApiError, NoMessagesError, InternalError
 
 logger = wrap_logger(logging.getLogger(__name__))
 
 
+def get_conversation(thread_id):
+    logger.debug("Retrieving conversation")
+
+    url = f'{current_app.config["BACKSTAGE_API_URL"]}/v1/secure-message/threads/{thread_id}'
+
+    response = requests.get(url, headers={'Authorization': _get_jwt()})
+
+    try:
+        response.raise_for_status()
+    except (HTTPError, RequestException):
+        logger.exception("Thread retrieval failed ")
+        raise ApiError(response)
+    logger.debug("Thread Retrieval successful")
```

Comment:
Could we include thread_id in the log messages?  It would be hard to investigate errors without specifics.

Reply:
@gavinleeedwards that is a debug level log, you would not find it in case of error. Do you want it actually to change the logging level in case ?

Reply:
I'd say that all errors should be exception level (which this one is) so that they are logged by default and will be picked up by any alerting that we may set up in future.  The log level for the successful retrieval is fine as debug.  Adding the thread_id to both logs would be useful though.